oauth-keycloak-authorizer/src/main/java/io/strimzi/kafka/oauth/server/authorizer/KeycloakRBACAuthorizer.java

```diff
@@ -598,7 +604,7 @@ private JsonNode fetchAuthorizationGrants(String token) {
             throw e;
         } catch (Exception e) {
             addGrantsHttpMetricErrorTime(e, startTime);
-            throw new RuntimeException("Failed to fetch authorization data from authorization server: ", e);
+            throw new ServiceException("Failed to fetch authorization data from authorization server: ", e);
```

Comment:
Could we get rd of the `catch (HttpException e)` block above?

Reply:
No. The exception handling in several places up the stack is different when HttpException occurs.